docs/devguide/run_time_configuration.rst

```diff
@@ -0,0 +1,2436 @@
+
+     OpenDDS::DCPS::TransportRegistry::instance()->bind_config("config_b", dp);
+
+Any Data Writers or Readers owned by this Domain Participant should now use the “``B``” side configuration.
+
+.. note:: When directly binding a configuration to a data writer or reader, the ``bind_config`` call must occur before the reader or writer is enabled.
+  This is not an issue when binding configurations to Domain Participants, Publishers, or Subscribers.
+  See Section :ref:`quality_of_service--entity-factory` for details on how to create entities that are not enabled.
+
+.. _run_time_configuration--transport-registry-example:
+
+Transport Registry Example
+==========================
+
+..
+    Sect<7.4.3>
+
+OpenDDS allows developers to also define transport configurations and instances via C++ APIs.
+The ``OpenDDS::DCPS::TransportRegistry`` class is used to construct ``OpenDDS::DCPS::TransportConfig`` and ``OpenDDS::DCPS::TransportInst`` objects.
+The ``TransportConfig`` and ``TransportInst`` classes contain public data member corresponding to the options defined below.
+This section contains the code equivalent of the simple transport configuration file described in .
+First, we need to include the correct header files:
+
+.. code-block:: cpp
+
+    #include <dds/DCPS/transport/framework/TransportRegistry.h>
+    #include <dds/DCPS/transport/framework/TransportConfig.h>
+    #include <dds/DCPS/transport/framework/TransportInst.h>
+    #include <dds/DCPS/transport/tcp/TcpInst.h>
+
+    using namespace OpenDDS::DCPS;
+
+Next we create the transport configuration, create the transport instance, configure the transport instance, and then add the instance to the configuration’s collection of instances:
+
+.. code-block:: cpp
+
+      TransportConfig_rch cfg = TheTransportRegistry->create_config("myconfig");
+      TransportInst_rch inst = TheTransportRegistry->create_inst("mytcp", // name
+                                                                 "tcp");  // type
+
+      // Must cast to TcpInst to get access to transport-specific options
+      TcpInst_rch tcp_inst = dynamic_rchandle_cast<TcpInst>(inst);
+      tcp_inst->local_address_str_ = "myhost";
+
+      // Add the inst to the config
+      cfg->instances_.push_back(inst);
+
+Lastly, we can make our newly defined transport configuration the global transport configuration:
+
+.. code-block:: cpp
+
+      TheTransportRegistry->global_config(cfg);
+
+This code should be executed before any Data Readers or Writers are enabled.
+
+See the header files included above for the full list of public data members and member functions that can be used.
+See the option descriptions in the following sections for a full understanding of the semantics of these settings.
+
+Stepping back and comparing this code to the original configuration file from , the configuration file is much simpler than the corresponding C++ code and has the added advantage of being modifiable at run-time.
```

Comment:
This seems to be missing something between "from" and `,`

Reply:
That's from the DevGuide, but I can fix it.